### PR TITLE
create johnbeatrice.md

### PR DESCRIPTION
**Pull request by johnbeatrice**
Raw.githack link to document:

https://raw.githack.com/johnbeatrice/johnbeatriceOLE.github.io/johnbeatrice-first-branch/#!pages/vi/profiles/johnbeatrice.md

### CODE REVIEW

**Comment by dcwebdev** [comment]
Hi, I'd like to unsubscribe from this thread, but the unsubscribe link
doesn't work for me. Could a project manager remove my email from the list,
please?

File: pages/vi/profiles/johnbeatrice.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# John Beatrice
+
+Location | Timezone | OS
+:--- | :--- | :---
+Greater Boston | EST | Ubuntu 18.04.3|
+
+## About Me
+Hi there, I`m a self-taught web developer transitioning careers, and I'm very excited to be a part of OLE! I've worked in a variety of industries before now but I feel I've found a great fit in tech and am super excited to work as a software engineer!
```
Comment (lmmrssa) [inline pages/vi/profiles/johnbeatrice.md:8]: "`" in this line will be an issue please use normal quote.

**Review by lmmrssa — CHANGES_REQUESTED** [review]

File: pages/vi/profiles/johnbeatrice.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Location | Timezone | OS
+:--- | :--- | :---
+Greater Boston | EST | Ubuntu 18.04.3|
+
+## About Me
+Hi there, I'm a self-taught web developer transitioning careers, and I'm very excited to be a part of OLE! I've worked in a variety of industries before now but I feel I've found a great fit in tech and am super excited to work as a software engineer!
+
+
+## Contact
+* __LinkedIn:__ https://www.linkedin.com/in/john-beatrice-2071a8218/
```
Comment (lmmrssa) [inline pages/vi/profiles/johnbeatrice.md:12]: please use markdown for the link. Right now it is browser which is showing it as link not the markdown

File: pages/robots/rbts-myplanet.md [hunk 3]
```diff
@@ -17,8 +17,8 @@ To get started vist Open Learing Exchange on GitHub to view all working
 [repositories](https://github.com/open-learning-exchange). Or jump straight to the [myPlanet](https://github.com/open-learning-exchange/myplanet) 
 repository's README file where you can find set up information. 
 
-## Usefull Links
```
Comment (lmmrssa) [inline pages/robots/rbts-myplanet.md:20]: This should be separate PR so please clean this PR just to put your profile file and create separate PR for this issue

**Review by lmmrssa — APPROVED** [review]

**Pull request merged**